.gitattributes

```diff
@@ -0,0 +1,52 @@
+* text=auto
+*.php text
+composer.lock merge=ours
+
+# Root
+/.gitattributes export-ignore
+/.travis.yml export-ignore
+/.github export-ignore
+/var export-ignore
+/bisect_phpunit.sh export-ignore
+/bootstrap.php export-ignore
```

Comment:
So if `/config.php-DEVELOPMENT` is commented out, why this one isn't? It's also needed for tests.

Reply:
This one is needed for kernel tests, but afaik not for solr tests which is the case for `config.php-DEVELOPMENT` or?

That said I opened this originally on 7.5, in this case we can aim to clean this up in master to not need this file anymore at all.

Reply:
> `boostrap.php` one is needed for kernel package tests itself, but not for solr bundle integration tests which only needs `config.php-DEVELOPMENT` afaik.
> 
> That said I opened this originally on 7.5, in this case we can aim to clean this up in master to not need this file anymore at all.

Nah, the point with Solr makes sense :+1: 
I'll try to revise test setup improvements for 8.0 anyway and then we can tackle it.